.gitignore

```diff
@@ -161,3 +161,10 @@ some_data.csv
 profile.dump
 .env.sh
 staticfiles
+trace.zip
+
+# Sandbox Files
+.personal/
+.sandbox/
+sandbox/
+sandbox.py
```

Comment:
EOF

Reply:
idk what EOF means, but you can put these exclusions in `.git/info/exclude` which isn't tracked.

Reply:
oh EOF meaning end of file needs a newline. Which is also true, haha, and wasn't true before this edit.
We can keep `trace.zip` here, but files specific to your developing style should be in your instance's exclude instead.
(id argue that for `some_data.csv` as well, but I missed that getting added)